server/public/media.css

```diff
@@ -1,25 +1,80 @@
 @media screen and (min-width: 320px) {
+  .leaflet{
+    bottom:7.0%
+ }
   .logo {
     width: 40%;
   }
-
-
+}
+@media screen and (min-width:375px){
+  .leaflet{
+    bottom:4.9%;
+ }
+ .panel{
+   top:69%;
+ }
+}
+@media screen and (min-width:414px){
+  .leaflet{
+    bottom:5.5%
+ }
+}
   /********  HORIZONTAL LAYOUT!  ****/
-  @media screen and (min-width: 568px) {
-    .logo {
-      width: 15%;
-    }
-
-    @media screen and (min-width: 736px) {
-      .logo {
-        width: 15%;
-      }
-
-      @media screen and (min-width: 768px) {
-        .logo {
-          width: 25%;
-        }
-      }
-    }
+@media screen and (min-width: 568px) {
```

Comment:
I would reduce the amount of media queries to match the number contained in bootstrap, and set your own queries to match those breakpoints. https://getbootstrap.com/docs/4.6/layout/overview/